crates/rooch-framework/sources/bitcoin_light_client.move

```diff
@@ -103,11 +103,42 @@ module rooch_framework::bitcoin_light_client{
         while(idx < vector::length(txoutput)){
             let txout = *vector::borrow(txoutput, idx);
             let outpoint = bitcoin_types::new_outpoint(txid, (idx as u32));
-            table::add(&mut btc_store.utxo, outpoint, txout);
+            add_utxo(btc_store, outpoint, txout);
             idx = idx + 1;
         }
     }
 
+    fun remove_utxo(btc_store: &mut BitcoinStore, outpoint: OutPoint){
+        if(table::contains(&btc_store.utxo, outpoint)){
+            let txout = table::remove(&mut btc_store.utxo, outpoint);
+            let script_pubkey = bitcoin_types::txout_script_pubkey(&txout);
+            //TODO should we record the address of the utxo?
+            let address_opt = bitcoin_address::from_script(script_pubkey);
+            if(option::is_some(&address_opt)){
+                let address = option::extract(&mut address_opt);
+                let balance = table::borrow_mut_with_default(&mut btc_store.balance, address, 0);
+                if (*balance > bitcoin_types::txout_value(&txout)){
```

Comment:
1. `>` Should it be `>=`? 
2. In addition, if the balance is not enough, should abort or just emit an event?

Reply:
1. fixed.
2. Record a comment; I will handle it with the UTXO miss.